tests/unit/test_docs.py

```diff
@@ -87,5 +113,7 @@ def test_folders_contain_index(data):
 
 @pytest.mark.parametrize("start_end", START_END_LINES, ids=[section["name"] for section in START_END_LINES])
 def test_docs_start_end_lines(start_end):
-    assert README_LIST[start_end["start_line"]] == start_end["start_value"]
-    assert README_LIST[start_end["end_line"]] == start_end["end_value"]
+    start_line_value = _get_start_line(start_end["name"])
+    end_line_value = _get_end_line(start_end["name"])
```

Comment:
Can you add one failure test as well? I feel like it hasn't always failed proactively.

Reply:
I added a test to that asserts a different end value doesn't match the README. I'm not sure if that is sufficient though. I'm not sure how to test a failure as no exceptions get raised when it "fails"